.azuredevops/scripts/create_ado_pipelines.sh

```diff
@@ -0,0 +1,52 @@
+repo_name=$1
+project_name=$2
+path_to_infrastructure_pipelines=infrastructure/pipelines
+path_to_mlops_pipelines=mlops/devops-pipelines
+
+cd $repo_name
+
+az pipelines folder create \
+    --path $repo_name \
+    --project $project_name
+
+az pipelines folder create
+    --path $repo_name/infrastructure \
+    --project $project_name
+
+az pipelines folder create
+    --path $repo_name/mlops \
+    --project $project_name
+
+mlops_pipeline_files=$(ls $path_to_mlops_pipelines)
+
+for file in $mlops_pipeline_files
+do
+    az pipelines create \
+        --name ${file%.*} \
+        --detect true \
+        --description "Automatically created pipeline for MLOps $file" \
+        --repository $repo_name \
+        --branch here-is-your-template \
```

Comment:
Can we make this branch naming "main"? What is this branch actually pointing to at the moment?